packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -96,9 +115,13 @@ export interface IPortfolio extends cdk.IResource {
   notifyOnStackEvents(product: IProduct, topic: sns.ITopic, options?: CommonConstraintOptions): void;
 
   /**
-   * Add a Resource Update Constraint.
+   * Force users to assume a certain role when launching a product.
+   *
+   * @param product A service catalog product.
+   * @param launchRole The IAM role a user must assume when provisioning the product.
+   * @param options options for the constraint.
    */
-  constrainTagUpdates(product: IProduct, options?: TagUpdateConstraintOptions): void;
+  setLaunchRole(product: IProduct, launchRole: iam.IRole, options?: CommonConstraintOptions): void;
```

Comment:
This comment has not been addressed...?

Reply:
Sorry this comment wasn't loading for me.  For one, I am worried about introducing a different API interaction amongst service catalog CDK.  Another issue is that in our sample code we don't have this but an end user will have a lot of specific policies listed for the role so I think that is normally something very explicit.  Otherwise users will have to addPolicies to that role which I am not very familiar with.

Reply:
Fair enough. We can always make it optional later.